app-modules/prospect/src/Filament/Widgets/ProspectStats.php

```diff
@@ -66,7 +66,7 @@ protected function getStats(): array
                 })),
             Stat::make('My Alerts', Cache::tags(['prospects', "user-{$user->getKey()}-prospect-alerts"])
                 ->remember("user-{$user->getKey()}-prospect-alerts-count", now()->addHour(), function () use ($user): int {
-                    return $user->prospectAlerts()->status(AlertStatus::Active)->count();
+                    return $user->prospectAlerts()->status(SystemAlertStatusClassification::Active)->count();
```

Comment:
Does this need to change based on the Feature flag to check the relationship rather than by ENUM now?